src/auth/auth.module.ts

```diff
@@ -0,0 +1,9 @@
+import { Module } from "@nestjs/common"
+import { AuthService } from "./auth.service"
+import { AuthController } from "./auth.controller"
+@Module({
+  imports: [],
+  providers: [AuthService],
+  controllers: [AuthController]
```

Comment:
Could you add an export line here? Like this: 
@Module({
  imports: [],
  providers: [AuthService],
  controllers: [AuthController],
  exports: [AuthService],
})